packages/react/src/transactions/adapters/TypedData.ts

```diff
@@ -1,7 +1,8 @@
-import { TypedDataDomain, TypedDataField } from 'ethers';
+import { Eip712TypedDataDomain, OmitTypename } from '@lens-protocol/api-bindings';
+import { TypedDataField } from 'ethers';
 
 export type TypedData = {
-  domain: TypedDataDomain;
+  domain: OmitTypename<Eip712TypedDataDomain>;
```

Comment:
Better type. Not strictly needed by the implementation but helped with the test helpers and it's better overall.